tron/kubernetes.py

```diff
@@ -621,6 +624,7 @@ def recover(self, task: KubernetesTask) -> None:
 class KubernetesClusterRepository:
     # Kubernetes config
     kubernetes_enabled: bool = False
+    kubernetes_non_retryable_exit_codes: Tuple[int, ...] = ()
```

Comment:
Again, I think we want a list.

Additionally, I think it might be nicer to allow this as `Optional` in the same way our other configs are set here (tho this would require us to set a default properly if we get `None` in `KubernetesCluster` above, similar to how default_volumes/pod_launch_timeout are set)